test/run/idl.mo

```diff
@@ -7,10 +7,10 @@ func deserUnit(x : Blob) : () = (prim "deserialize" : Blob -> ()) x;
 func serNats(x: Nat, y: Nat, z: Nat) : Blob = (prim "serialize" : (Nat,Nat,Nat) -> Blob) (x,y,z);
 func deserNats(x: Blob) : (Nat, Nat, Nat) = (prim "deserialize" : Blob -> (Nat,Nat,Nat)) x;
 
-func serText(x: Text) : Blob = (prim "serialize" : Text -> Blob) x;
+func serText(x: Text) : Blob = (prim "serialize" : Text -> Blob) (x,);
```

Comment:
Now learned that the parsing of `x` and `(x,)` is the same by virtue of [our parser ignoring these parens for single member tuples](https://github.com/dfinity/motoko/blob/cacf7742bd9e2c40f985fd29049c8c28847ac822/src/mo_frontend/parser.mly#L556).